elastalert/schema.yaml

```diff
@@ -495,7 +495,7 @@ properties:
   ### Slack
   slack_webhook_url: *arrayOfString
   slack_username_override: {type: string}
-  slack_channel_override: {type: string}
+  slack_channel_override: *arrayOfString
```

Comment:
`slack_channel_override` can be either a single string or many:

https://github.com/jertel/elastalert2/blob/c982b9489be8f0856a943e3bc7bd9d532b284701/elastalert/alerters/slack.py#L23